nservicebus/sqlserver/deployment-options.md

```diff
@@ -45,9 +45,9 @@ The transport will route messages to destination based on the configuration. If
 
 ### Multi-instance
 
-WARNING: The *multi-instance* option won't be directly supported in Versions 4 and higher. Instead, a guidance will be provided on how to achieve a similar result using built-in SQL Server features. However, in Versions 4 and higher the *multi-catalog* option will be directly supported.
+WARNING: The *multi-instance* option won't be supported in Versions 4 and higher. Version 3 supports *multi-instance* mode only with DTC. Instead, a guidance will be provided on how to achieve a similar result using built-in SQL Server features. However, in Versions 4 and higher the *multi-catalog* option will be directly supported.
```

Comment:
I've also removed the guidance promise. When it will be ready we can link it here.